.github/workflows/mac_build_test.yml

```diff
@@ -47,7 +47,7 @@ jobs:
       - name: Initial setup
         shell: bash -l {0}
         run: |
-          brew install eigen hdf5
+          brew install --force eigen hdf5
```

Comment:
I think the new HDF5 comes with a new pkgconf, which is causing the linking issue.

We could consider running:

```
brew link --overwrite pkgconf
or
brew reinstall pkg-config
```

Reply:
I don't know enough about brew to know in which order this would come relative to the lines we already have?